src/test/java/no/unit/nva/institution/proxy/handler/NestedInstitutionHandlerTest.java

```diff
@@ -104,16 +103,18 @@ void testAnInstitutionCanBeNestedFromLiveData() throws IOException {
     @DisplayName("Tests that a single unit can be nested from live data")
     @Test
     @Tag("online")
-    void testSingleUnitCanBeNestedFromLiveData() throws IOException {
+    void testSingleUnitCanBeNestedFromLiveData() throws Exception {
         TestContext context = new TestContext();
 
         NestedInstitutionHandler nestedInstitutionHandler =
             new NestedInstitutionHandler(environment, new CristinApiClient());
-        NestedInstitutionRequest request =
-            new NestedInstitutionRequest("https://api.cristin.no/v2/units/194.63.1.20", "en");
+
+        InputStream input = Try
```

Comment:
Update obsolete online tests to fit the new query format. These are very old queries that  were sending a body for the query. The current query format sends query parameters